docs/platform/release-notes/tizen-5-5-m2.md

```diff
@@ -0,0 +1,641 @@
+  - SVG elements and their properties have been extended.
+  - CSS animation feature has been added.
+  - WebSocket feature has been added.
+  - Animated GIF has been supported.
+- JS Engine (ES6 Features)
+  - Class type has been added.
+  - Pattern feature has been added.
+  - Proxy feature has been added.
+  - Iterator operations have been added.
+  - Extended parameter has been added.
+  - Generator has been added.
+- Support for headless profile has been added.
+- Performance and Memory Optimization
+  - Rendering Engine
+    - Direct mode has been enabled on EvasGL backend.
+    - CompositorGL has been fixed to enhance performance.
+    - Multithreaded image decoding feature has been added.
+    - OpenGL Texture is re-used to reduce loading time.
+  - JS Engine
+    - Lexer objects have been allocated on stack memory rather than heap memory.
+    - The initialization process of an interpreter has been simplified.
+
+### Fixes
+
+- Bug fixes
+  - Layout bugs have been fixed.
+  - Memory leaks have been fixed.
+  - Crashes have been fixed.
+
+
+## Tizen .NET
+
+### New and changed features
+
+- .NET Core (Runtime)
+  - .NETCore has been upgraded from 3.0.0-preview2 to 3.0.0.
+- Xamarin.Forms
+  - Xamarin.Forms 4.2.0, which is latest stable version has been supported.
+- Tizen.CircularUI
+  - Wearable UI extension (Tizen.Wearable.CircularUI) 1.3.0 has been supported.
+- TizenFX (C# Device API) API Level 6
+  - New Application API set for component ID has been added.
+  - New ComponentBased API set has been added.
+  - New AudioManager API set for audio ducking has been added.
+  - New PackageManager API set to get dependency information has been added.
+  - New MachineLearning Inference API set has been added.
+  - New MediaContent API set to add face information into the database manually has been added.
+  - Not used filter keywords of MediaContent has been deprecated.
+  - New AudioChannel and SampleType of AudioIO have been added.
+  - New ImageUtil API set to extract thumbnail has been added.
+  - New MediaPlayer API set to export video frames and audio PCM data has been added.
+  - New MediaPlayer API set to cancel preparing has been added.
+  - New MediaPlayer API set for audio codec and offload has been added.
+  - New Multimedia Remote API set for subtitles and 360 mode has been added.
+  - New Vision API set for detection and classification using the inference engine has been added.
+  - New UI component set of NUI has been added.
+  - Tizen.NUI.UIComponents module has been deprecated.
+  - New NUI API set for window creation and rotation has been added.
+  - New SystemSettings API set to get rotary event enabled has been added.
+  - New VoiceControlManager API set has been added.
+
+### Known Issues
+
+- Xamarin.Forms
+  - For more information on the list of limitations, see [here](https://developer.tizen.org/development/api-reference/.net-application/current-xamarin.forms-limitations).
```

Comment:
Please change the link to ../../application/dotnet/api/xamarin-forms-limitations.md

Reply:
will accept